tests/UI/campaigns/functional/FO/classic/06_homePage/01_checkSlider.ts

```diff
@@ -61,8 +61,6 @@ describe('FO - Home Page : Check slider', async () => {
     await testContext.addContextItem(this, 'testIdentifier', 'checkSliderURL', baseContext);
 
     const currentURL = await homePage.getSliderURL(page);
-    await expect(currentURL)
-      .to.contains('www.prestashop-project.org')
-      .and.to.contains('homeslider&utm_campaign=back-office-EN&utm_content=download');
+    await expect(currentURL).to.contains('www.prestashop-project.org');
```

Comment:
@nesrineabdmouleh Why this change ? Isn't it a regression ?

Reply:
I don't know why it's changed but I think it's not a problem it's just a link for the slide